stablehlo/reference/Ops.cpp

```diff
@@ -560,6 +560,13 @@ SmallVector<InterpreterValue> eval(
       auto operand = scope.findTensor(realOp.getOperand());
       auto result = evalRealOp(operand, realOp.getType());
       scope.add(realOp.getResult(), result);
+    } else if (auto recvOp = dyn_cast<RecvOp>(op)) {
+      auto token = scope.findToken(recvOp.getToken());
+      ChannelId channelId = 0;
```

Comment:
How does the spec go about handling default values? Looks like ChannelId has a default value of 0 in the interpreter.. should this be documented elsewhere, or do we tend to leave default values out of the spec?

Reply:
The spec does not go into detail about default values as this is implementation specific.